### ISSUE TITLE
Error notification on recipient_country_code being null is unneeded

### ISSUE THREAD

**Issue by Popolechien** [kiwix/cardshop#224]
This is fine in and of itself but misleading: if I create an order (in this case 5fdcb9c14eb) and the `country` field in the recipient's address in not filled, I will get an error message
![Capture d’écran 2020-12-18 à 15 18 26](https://user-images.githubusercontent.com/21675885/102624692-a662b380-4144-11eb-8d21-23e21c90db85.png)

Which would lead me to correct the address and add a country, but in the meantime I did receive a confirmation email saying that the order had been accepted. 
Bottom line: `recipient_country_code` should be allowed to be `null`

**Comment by Popolechien** [comment]
weird duplicate as I got an error message from Github after creating #223
